src/html/settings.html

```diff
@@ -306,6 +314,28 @@ <h3>Pomodoro settings</h3>
           </div>
         </section>
       </div>
+      <div class="tab" data-tab="website-blocking">
+        <section>
+          <div class="headline">
+            <h3>Website blocking</h3>
+          </div>
+          <p>To avoid distractions, you can specify a list of websites to block when time-tracking is on.</p>
+          <div class="checkbox field enable-website-blocking">
+            <input type="checkbox" id="enable-website-blocking">
+            <label for="enable-website-blocking">
+              <span class="setting">Enable website blocking</span>
+              <span class="description">If enabled, visiting websites from the website blocking list will be disallowed while you're tracking time</span>
+            </label>
+          </div>
+          <div class="textarea field">
+            <label for="website-blocking-list">
+              <span class="setting">Website blocking list</span>
+              <span class="description">Enter the list of domains you would like to block, each one on a new line</span>
+            </label>
+            <textarea id="website-blocking-list" rows="3"></textarea>
```

Comment:
The colour of the text inside the textarea is some sort of faded grey - looks a bit like it's disabled or a placeholder. Can we have it in some other colour?